Keas.Mvc/ClientApp/components/People/PeopleTable.tsx

```diff
@@ -68,7 +86,7 @@ export default class PeopleTable extends React.Component<IProps, {}> {
 
     const initialState: Partial<TableState<any>> = {
       sortBy: [{ id: 'name' }],
-      pageSize: ReactTableUtil.getPageSize(),
+      pageSize: ReactTableUtil.getPageSize()
```

Comment:
Didn't we decide to go with trailing commas?

Reply:
If we did, we didn't codify it in the peaks config file